test/elixir_bench_web/schema/schema_test.exs

```diff
@@ -38,26 +21,45 @@ defmodule ElixirBenchWeb.SchemaTest do
         "data" => %{
            "repos" => [
              %{
-               "name" => "ecto",
-               "owner" => "tallysmartins",
-               "slug" => "tallysmartins/ecto"
+               "name" => repo.name,
+               "owner" => repo.owner,
+               "slug" => repo.owner <> "/" <> repo.name
+             },
+             %{
+               "name" => another_repo.name,
+               "owner" => another_repo.owner,
+               "slug" => another_repo.owner <> "/" <> another_repo.name
              }
            ]
          }
        }
 
       {:ok, resp} = context.conn
-        |> post("/api", AbsintheHelpers.query_skeleton(query))
-        |> Map.get(:resp_body)
-        |> Antidote.decode
+        |> execute_query(query)
```

Comment:
Also nothing big, but I think the newer elixir format way (you can execute `mix format file`) to do these

```elixir
bla =
  chain_start
  |> chain
  |> chain2
```

Please don't format the whole file, it'd be hard to review :)

Reply:
Nice! Didn't know about this. When I set the style formatter I can do a major refactoring.